pkg/api/filters/filter.go

```diff
@@ -0,0 +1,40 @@
+package filters
+
+import "net/http"
+
+//Filter is a wrapper around a http.HandlerFunc.
```

Comment:
_Total_ style nit. Can we put a space after `//`. Applicable in many other spots, too. If you say no, we can still be friends, so don't feel like you have to.